chromadb/test/property/test_embeddings.py

```diff
@@ -401,3 +403,29 @@ def test_delete_empty_fails(api: API):
     coll.delete(ids=["foo"], where={"foo": "bar"})
     coll.delete(ids=["foo"], where_document={"$contains": "bar"})
     coll.delete(ids=["foo"], where_document={"$contains": "bar"}, where={"foo": "bar"})
+
+
+@given(supported_types=st.sampled_from([np.float32, np.int32, np.int64, int, float]))
+def test_autocasting_validate_embeddings_for_compatible_types(
+    supported_types: list[Type[Any]], caplog: pytest.LogCaptureFixture
+) -> None:
+    embds = strategies.create_embeddings(10, 10, supported_types)
+    validate_embeddings(embds)
```

Comment:
this just tests the method runs? This is fine, but ideally would assert the list values are with atol